pan/types.pan

```diff
@@ -1042,3 +1042,18 @@ type string_search_path = string_trimmed with {
     if (dot_count > 1) error('The search path "%s" contains more than one reference to the working directory.', SELF);
     true;
 };
+
+
+@documentation{
+Returns true if the string is a valid octal mode/mask
+}
+function is_octal_mode = {
```

Comment:
no need for a function imho. simply make a type with the regexp.

Reply:
done